test/getsettest.c

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I doubt that we should be trying to exercise the entire SSL API in the same unit test, but ok. If so then you're _definitely_ going to need the unit test framework :)

i_use_underscores_to_separate_words_test would make it easier to my eyes.